### ISSUE TITLE
Import from disk - content shown to be available for import when it is not

### ISSUE THREAD

**Issue by nobicycle** [learningequality/kolibri#3269]
### Instructions
**Instructions:**
Import content from a usb flash disk on Linux
### Observed behavior
The channels display seems OK:
http:/<lan>/management/device/#/content/wizard/availablechannels
It seems to show the correct mounted partition  where the KOLIBRI_DATA directory exists.
/mnt
It shows the correct number of channels exported.
7 channels available
![import from mnt](https://user-images.githubusercontent.com/1372447/36741070-20e3985c-1c1f-11e8-8840-b77427aa0932.jpg)

However if a channel is selected, eg. Khan Academy, it does not show the content actually available on the flash disk. It seems to be displaying what one would see from the Central Server.
![khan academy - shows nothing](https://user-images.githubusercontent.com/1372447/36741236-8cdc1fde-1c1f-11e8-8729-fb7c573d7001.jpg)

-->

…

### Expected behavior

During import from flash disk, the content actually available on the flash disk should be shown. It seems to be displaying what one would see from the Central Server.

### User-facing consequences
Cannot transfer any content

…

### Errors and logs
The flash disk partition was mounted as follows
sudo mount /dev/sdb3 /mnt

```
$ lsblk 
NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sda      8:0    0 232.9G  0 disk 
├─sda1   8:1    0   512M  0 part /boot
└─sda2   8:2    0 232.4G  0 part /
sdb      8:16   1  28.9G  0 disk 
├─sdb1   8:17   1   160M  0 part 
├─sdb2   8:18   1     5G  0 part 
└─sdb3   8:19   1  23.7G  0 part /mnt

-->]$ ls /mnt -l
total 5732
...
drwxr-xr-x 3 me me    4096 Feb 28 00:14 KOLIBRI_DATA
...
```

### Context
Tell us about your environment, including:
 * Kolibri version kolibri-v0.7.1.pex
 * Operating system
Linux kb.barana.net 4.14.21-1-MANJARO #1 SMP PREEMPT Fri Feb 23 00:01:40 UTC 2018 x86_64 GNU/Linux
 * Browser
Vivaldi 
1.14.1077.41 (Stable channel) (64-bit)

This was a fresh install. No content from the Central Server had been fetched.

**Comment by benjaoming** [comment]
Hi @nobicycle 

Thanks for reporting!

> However if a channel is selected, eg. Khan Academy, it does not show the content actually available on the flash disk. It seems to be displaying what one would see from the Central Server.

This could be a real issue! But can I ask you just to check that file permissions are set correctly?

Judging from your other issue, you are running `kolibri` as a systemd service with a user account called `kolibri`. In this case, you should run this command to ensure that everything belongs to that account:

    sudo chown kolibri.kolibri /mnt/KOLIBRI_DATA/

**Comment by rtibbles** [comment]
Note that this is currently 'expected behaviour' but we are working out how to better display the actually available files in the interface.

**Comment by indirectlylit** [comment]
this is being addressed in ongoing changes to 0.13.0
